TourGuide/src/main/java/tourGuide/beans/AttractionBean.java

```diff
@@ -0,0 +1,19 @@
+package tourGuide.beans;
+
+import java.util.UUID;
+
+public class AttractionBean extends LocationBean {
+
+    public final String attractionName;
```

Comment:
could be just a `name`, because `attractionBean. attractionName` sounds weird, wdyt?